website/templates/profile.html

```diff
@@ -209,15 +210,11 @@ <h1 class="page-header text-capitalize">{{ user.username }}</h1>
             {% endif %}
 
             <div class="status">
-                <form name="_xclick" action="https://www.paypal.com/cgi-bin/webscr" method="post">
-                    <input type="hidden" name="cmd" value="_xclick">
-                    <input type="hidden" name="business" value="{{ project.paypal|default:" coderbounty@gmail.com" }}">
-                    <input type="hidden" name="item_name" value="tip for {{ user.username }} on {% env 'DOMAIN_NAME' %}">
-                    <input type="hidden" name="currency_code" value="USD">
-                    <button type="submit" class="btn btn-danger" style="color: black;">
-                        <i class="fa fa-money fa-lg"></i> {% trans 'Send a tip' %}
-                    </button>
-                </form>
+                <button type="button" id="openModal" class="btn btn-danger" style="color: black;">Send a Tip</button>
+                <dialog id="dialog">
+                    <img id="qr-image" src="{% url 'generate_bch_qr' username=request.user.username %}" alt="QR Code">
+                    <p id="bch-address" style="color: aliceblue;">BCH: {{ request.user.userprofile.crypto_address }}</p>
```

Comment:
This is still current users

Reply:
can you elaborate on what you're looking for I am really confused

Reply:
This should be the address of the user, who we are requesting the address of not the current request.user

Reply:
Looks like it

Reply:
Commited the change please check it out